### PR TITLE
Redirect the user to their previous page after login/logout.

### PR DESCRIPTION
**Pull request by sizzlemctwizzle**
Works for me locally. I'm not sure if it will work in production. It uses the "Referer" header and I'm not sure if Nodejitsu handles that properly.

### CODE REVIEW

**Comment by sizzlemctwizzle** [comment]
Only one way to know. Deploying...

**Comment by sizzlemctwizzle** [comment]
Tested and it apparently works!

File: controllers/index.js [hunk 1]
```diff
@@ -151,13 +152,30 @@ exports.home = function (aReq, aRes) {
...
 
+// Get the referer url for redirect after login/logout
+function getRedirect(aReq) {
+  var referer = aReq.get('Referer');
+  var redirect = '/';
+
+  if (referer) {
+    referer = url.parse(referer);
+    if (referer.hostname === aReq.hostname) {
+      redirect = referer.path;
```
Comment (Martii) [inline controllers/index.js]: Was there a reason why you chose to omit search query and hash e.g. `?yadda=ying#comment-someHash`?

**Comment by Martii** [comment]
Nice addition. :)

**Comment by Martii** [comment]
Btw testing if referer is blocked momentarily to see what happens.

**Comment by Martii** [comment]
Blocked `referer` header redirects to `/` ... so perfect.

**Pull request merged**